src/navigation/DrawerNavigator.tsx

```diff
@@ -96,20 +95,15 @@ export const DrawerNavigator = () => {
         />
       )}
       screenOptions={{
-        drawerPosition:"right",
-        headerShown: false
+        drawerPosition: 'right',
+        headerShown: false,
+        drawerStyle: {
+          width: device.width > device.height ? device.height * 0.8 : device.width * 0.8
+        }
       }}
-      drawerStyle={styles.drawerContainer}
-      drawerType={device.platform === 'ios' ? 'slide' : 'front'}
```

Comment:
yes its not needed anymore since it defaults to slide on iOS and front on other platforms.
https://reactnavigation.org/docs/drawer-navigator/#drawertype